README.md

```diff
@@ -1,3 +1,25 @@
+## Requirements
+
+* scrapy>=2.5.1
+* zyte-api>=0.1.2
+* twisted>=21.7.0
+
+If you are deploying on Scrapy Cloud, then add the below packages to your requirements.txt file.
+
+* zyte-api>=0.1.2
+* twisted>=21.7.0
```

Comment:
These dependencies aren't needed once `scrapy-zyte-api` and `scrapy` is installed.

Reply:
Yea sure I will remove this part and leave the below part as it is.

git+https://github.com/scrapy-plugins/scrapy-crawlera-fetch.git

Reply:
I have removed it since we have published it in PIP.